com.wiz.Note.appdata.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.wiz.Note</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0+</project_license>
+  <name>Wiz Note</name>
+  <summary>An excellent PKM(personal knowledge management) desktop environment based on cloud usage</summary>
+  <description>
+    <p>
+      The project is based on Qt, aimed to provide an excellent PKM(personal knowledge management) desktop environment based on cloud usage. 
+    </p>
+    <p>
+      PKM should be an very important thing cross through one person's life, it's unwise to stick yourself to a fixed service provider or jump around and leave your collected info/secrets behide. PKM should be the same as your mind, fly over the ocean but never splash the waves.
+    </p>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Wiz.</p>
+  </description>
+  <launchable type="desktop-id">com.wiz.Note.desktop</launchable>
+  <categories>
```

Comment:
If the categories are already on the desktop file, you don't need to set them here too.